source/_components/lightwave.markdown

```diff
@@ -44,10 +44,16 @@ lightwave:
       name: Phone socket
     R2D4:
       name: Torch socket
+    registration:
+      name: Registration
 ```
-
+### `host`
```

Comment:
```suggestion
### {% linkable_title Host %}
```

Reply:
OK - I will investigate as I need to understand what your suggestion actually does.

Reply:
Makes it a linkable title, so you can make/copy a link that directs to this specific section.

Reply:
https://developers.home-assistant.io/docs/en/documentation_standards.html